acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -44,3 +46,25 @@
     end
   end
 end
+
+step 'Install build dependencies on master' do
+
+ MASTER_PACKAGES = {
+    :redhat => [
+      'git',
+      'java-1.8.0-openjdk-devel',
+    ],
+  }
+  install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+end
+
+step 'Confirm SSH Forwarding Enabled on master' do
+  ssh_auth_sock = on(master, 'echo $SSH_AUTH_SOCK').stdout.chomp
+  if ssh_auth_sock.nil? || ssh_auth_sock.empty?
+    fail_test('SSH forwarding not configured properly - check ~/.ssh/config for ForwardAgent settings')
+  end
+end
+
+step 'Allow git clones via ssh to "unknown" host at github.com; required to clone private repos.' do
```

Comment:
None. I think it refers to clj-pxp-puppet , but I removed the checkout of that repo as it's not currently used. I'll correct this comment.